lnbits/core/templates/core/wallet.html

```diff
@@ -62,6 +71,15 @@ <h3 class="q-my-none">
                 style="opacity: 0.75"
               ></span>
             </div>
+            <q-btn
+              @Click="mobileSimple = !mobileSimple"
+              class="float-right lt-md"
+              size="sm"
+              flat
+              style="color: primary"
```

Comment:
in other places this is used:
```css
color="primary"
```

Reply:
addressed in a47860df3f36a40862a1b85c33525e359b79f4e1